tests/services/FileLogger_Tests.cpp

```diff
@@ -20,6 +20,7 @@ TEST_CLASS(FileLogger_Tests)
 {
 private:
     const std::wstring mockLogFileName = L".\\RACache\\RALog.txt";
+    const std::wstring mockOldLogFileName = L".\\RACache\\RALog-old.txt";
```

Comment:
If you want you can make these `static constexpr`.

Reply:
Apparently I can't.
```
Error 	E2402	a constexpr variable must have a literal type or a reference type	RA_Integration.Tests
Error	C2131	expression did not evaluate to a constant	RA_Integration.Tests
```	
However, I can make a `constexpr` point to the string itself:
```
    static constexpr const wchar_t* mockLogFileName = L".\\RACache\\RALog.txt";
```
But then a new string object is created every time it's used. The current implementation creates a single string object for the life of the test fixture.